pom.xml

```diff
@@ -27,6 +27,7 @@
     <lucene.version>3.6.2</lucene.version>
     <powermock.version>1.5.5</powermock.version>
     <logback-classic.version>1.1.2</logback-classic.version>
+    <bouncycastle.version>1.52</bouncycastle.version>
```

Comment:
Current version is the recently released 1.53 , which has some compatibility fixes that might be of interest to the server:
```  The provider has also been updated to reflect changes in JDK 1.8 which broke X509Certificate.hashCode() and X509Certificate.verify(PublicKey, Provider) ``` 
from: http://bouncycastle.org/latest_releases.html